docker-compose.yml

```diff
@@ -1,19 +1,9 @@
 version: '3'
 services:
   windbreaker:
+    # set image tag for builds
+    image: windbreaker/windbreaker
     build: .
-    command: 'npm start'
-    depends_on:
-      - postgres
-      - rabbitmq
-    volumes:
-      - .:/usr/windbreaker
-      - /usr/windbreaker/node_modules
-    environment:
-      SERVICE_ENVIRONMENT: 'docker'
-  test:
-    build: .
-    command: 'npm run test:docker:ci'
```

Comment:
I removed the test container since it was pretty much a clone of the windbreaker container. For running tests, we use the `windbreaker` container but the test command on it instead. This also made it so that we don't explicitly have to run `docker-compose build windbreaker` in the travis config.